memgpt/schemas/source.py

```diff
@@ -27,5 +27,28 @@ class Source(BaseSource):
     name: str = Field(..., description="The name of the source.")
 
 
+class SourceQuery(BaseSource):
+    id: Optional[str] = Field(None, description="The ID of the source.")
+    name: Optional[str] = Field(None, description="The name of the source.")
+
+
+class SourceIdQuery(BaseSource):
```

Comment:
@sarahwooders is `get_source_id()` needed? or why do we have both `get_source_id()` and `get_source()`?

also is this pydantic model needed since it's a singleton is it better to just unpack it and have `name` in the signature for server.py?